packages/cli/src/__tests__/commands/Generate.test.ts

```diff
@@ -49,18 +49,7 @@ describe('using cli', () => {
 
         ✔ Generated Prisma Client (v0.0.0) to ./generated/client in XXXms
 
-        Start using Prisma Client in Node.js (See: https://pris.ly/d/client)
-        \`\`\`
-        import { PrismaClient } from './generated/client'
-        const prisma = new PrismaClient()
-        \`\`\`
-        or start using Prisma Client at the edge (See: https://pris.ly/d/accelerate)
-        \`\`\`
-        import { PrismaClient } from './generated/client/edge'
-        const prisma = new PrismaClient()
-        \`\`\`
-
-        See other ways of importing Prisma Client: http://pris.ly/d/importing-client
+        Start using Prisma Client (See: https://pris.ly/d/client)
```

Comment:
Does the link lead to a correct place now? Would it make sense for it to point to https://www.prisma.io/docs/orm/prisma-client/setup-and-configuration/introduction instead?

Reply:
So we have the following link `http://pris.ly/d/importing-client` for that. As far as I understood from @apolanc, we want to link to `https://pris.ly/d/client` and we can look at streamlining these pages a little bit so it's easier to navigate to importing new clients.

Reply:
I thought we were pointing to http://pris.ly/d/importing-client as a way to get rid of /client and /client/edge scenarios. What do you @SevInf and @Druue think of rephrasing this to:

`Start by importing your prisma client (See:  http://pris.ly/d/importing-client)`

Which is the next action if you run prisma migrate for the first time.

Reply:
Sounds good to me